trimChamber.py

```diff
@@ -86,8 +86,10 @@
             "-s%d"%(options.slot),
             "-g%d"%(options.gtx),
             "--filename=%s"%(filename0),
-	        "--vfatmask=%i"%(options.vfatmask)]
-           )
+            "--vfatmask=%i"%(options.vfatmask),
```

Comment:
This is definitely the wrong way to do this, since there is no expectation that the `vfatmask` will be the same for all chambers under consideration, probably best would be to have it stored in the `chamber_info` file per chamber